operators/dynatrace-operator/0.10.0/manifests/dynatrace-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1733 @@
+          - create
+        - apiGroups:
+          - ''
+          resources:
+          - secrets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - delete
+        - apiGroups:
+          - ''
+          resources:
+          - events
+          verbs:
+          - list
+          - create
+        - apiGroups:
+          - ''
+          resources:
+          - services
+          verbs:
+          - create
+          - update
+          - delete
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - networking.istio.io
+          resources:
+          - serviceentries
+          - virtualservices
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - update
+          - create
+        serviceAccountName: dynatrace-operator
+      - rules:
+        - apiGroups:
+          - ''
+          resources:
+          - services
+          - configmaps
+          - secrets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+        - apiGroups:
+          - ''
+          resources:
+          - pods
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - dynatrace.com
+          resources:
+          - dynakubes
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - ''
+          resources:
+          - events
+          verbs:
+          - list
+          - create
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - update
+          - create
+        - apiGroups:
+          - apps
+          resources:
+          - daemonsets
+          verbs:
+          - list
+          - watch
+        serviceAccountName: dynatrace-webhook
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - monitoring
+  - dynatrace
+  - oneagent
+  - activegate
+  - observability
+  links:
+  - name: Dynatrace Operator
+    url: https://github.com/Dynatrace/dynatrace-operator
+  - name: OpenShift Monitoring Info
+    url: https://www.dynatrace.com/technologies/openshift-monitoring
+  maintainers:
+  - email: support@dynatrace.com
+    name: Dynatrace LLC
+  maturity: alpha
+  provider:
+    name: Dynatrace LLC
+    url: www.dynatrace.com
+  version: 0.10.0
```

Comment:
You are missing replaces
```
replaces: dynatrace-operator.v0.9.1
```

Reply:
@mvalarh we have defined olm.skipRange instead of replaces, works fine here https://github.com/redhat-openshift-ecosystem/certified-operators/pull/1605

Reply:
Hi, any answer or feedback on this?

Reply:
I was looking at https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/how-to-update-operators.md#skiprange and i see that one have to define semver without `v`. Can you try this? Also maybe it is  needed to put value in '' as string